api/napi/package.json

```
@@ -19,7 +20,8 @@
   },
   "scripts": {
     "artifacts": "napi artifacts",
-    "build": "napi build --platform --release",
+    "compile": "esbuild index.ts --bundle --loader:.node=copy --format=cjs --platform=node --outfile=index.js",
```

Instead of copying the napi generated .node files, I think it's easier to tell esbuild that they are externals that can remain untouched.

```suggestion
    "compile": "esbuild index.ts --bundle --external:*.node --format=cjs --platform=node --outfile=index.js",
```